plugin.go

```diff
@@ -252,7 +287,7 @@ func (p *ormPlugin) generateMapFunctions(message *generator.Descriptor) {
 		}
 		p.generateFieldMap(message, field, false)
 	}
-	p.P(`return to`)
+	p.P(`return to, err`)
 	p.P(`}`)
```

Comment:
If the last field of a message `is not an ORMable message type` we have a compilation error
```console
 58:6: expected '(', found 'IDENT' ConvertACLFromORM (and 2 more errors)
--gorm_out: protoc-gen-gorm: Plugin failed with status code 1.
```

That is caused by missing `}` symbol for the `for` statement.

Reply:
Good catch. Specifically for repeated nonORMable fields. I'll fix that.

Reply:
Also missed the repeated UUID and Timestamp cases. Fixing those, too.